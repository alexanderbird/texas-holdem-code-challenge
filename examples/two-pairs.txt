2H 4C 6S 8D TC
Thebault 8C TS
Bren 6D 2D
Orran 8H 6C
Shela 8S TH
Lemuel 6H 4S
Cheater 6H 4D QC 
