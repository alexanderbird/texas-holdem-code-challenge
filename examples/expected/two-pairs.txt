2 Thebault Two Pair 10 8
2 Shela Two Pair 10 8
3 Orran Two Pair 8 6
4 Cheater Two Pair 6 4 (kicker Queen)
5 Lemuel Two Pair 6 4 (kicker 10)
6 Bren Two Pair 6 2
